src/App.js

```diff
@@ -16,6 +16,7 @@ const App = () => {
         <h1>
         Brassicaceae
         </h1>
+        <NavBar />
```

Comment:
Nice, we got a nav bar added! It looks like you have a Brassicaceae in the NavBar. Can we remove the h1 from and brassicaseae from this file? (App.js)

Reply:
Sure we can. I will remove it during my next PR, maybe tomorrow